configure

```diff
@@ -4545,7 +4545,7 @@ define_in_config_if_yes "$_16bit" 'USE_RGB_COLOR'
 #
 # Check whether High resolution graphics support is requested
 #
-define_in_config_if_yes "$_highres" 'USE_HIGHRES'
```

Comment:
The main "issue" I see with this change is that _highres is used as a positive filter in configure.engine file (engine declares it needs highres) while in the config.h it is used as a negative filter (defines a disable).